.gitignore

```diff
@@ -8,5 +8,8 @@
 /spec/reports/
 /tmp/
 
+# rubymine
+.idea
+
```

Comment:
Is that necessary? It looks like an accidental addition that is not part of this PR.